app/assets/stylesheets/top_bar.scss

```diff
@@ -0,0 +1,67 @@
+@import "colors";
+$topbar-height: 4rem;
+
+nav.top-bar,
+nav.tab-bar,
+.contain-to-grid {
+  background: $dark-blue;
+  height: $topbar-height;
+
+  .top-bar-section li:not(.has-form) a:not(.button) {
+    background: $dark-blue;
+  }
+
+  .top-bar-section .dropdown li:not(.has-form):not(.active) {
+    & > a:not(.button) {
+      background: $pink;
+    }
+
+    & > a:not(.button):hover {
+      background: darken($pink, 10%);
+    }
+
+    &:not(:last-child) a {
+      border-bottom: 0.1rem solid $white;
+    }
+  }
+
+
+  .top-bar-section .has-dropdown > a {
+    line-height: 3.7rem;
+    &:after {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4